app/views/order_details/order_file.html.haml

```diff
@@ -22,7 +22,8 @@
         %td
           %p
             %b Download and complete the template
-          %p.document= link_to "#{@order_detail.product.name} &mdash; Order Form Template".html_safe, @order_detail.product.stored_files.template.first.file.url
+          %p.document
+            = link_to "#{@order_detail.product.name} &mdash; Order Form Template".html_safe, product_file_path(@order_detail.product.stored_files.template.first)
```

Comment:
I'd like to change the route so it's not scoped under facility management. Regular users would see the link when scoped to an order, but that wouldn't make sense in the context of an admin managing a facility. Any suggestions?